PRC.owl

```diff
@@ -301,11 +301,13 @@
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000002">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000001"/>
-        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atom and partially positive nitrogen atom in the polypeptide backbone of amino acids and the specific 3-dimensional structure that results.</obo:IAO_0000115>
+        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in one or more amino acid chains and the specific 3-dimensional structure that results. Beta sheets in particular may involve the same or separate amino acid chains, whereas most other secondary structures involve only a single amino acid chain.</obo:IAO_0000115>
```

Comment:
A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in the backbones of one or more amino acid chains and the specific 3-dimensional structure that results.